### DISCUSSION TITLE
Pivot Sort

### DISCUSSION THREAD

**Discussion by RoYoMi** [dfinke/ImportExcel#1306] [Q&A]
I have created a pivot table but it don't see a way to make it sort the output by the Grand Total field.

In Excel I can manually right click the a total field and choose sort.

Here is my setup:
```
 $Excel = $Output.RawData `
       | Select-Object @{n="Date"; e={$_.Date | Get-Date}}, @{n="DateYM"; e={$_.Date | Get-Date -Format "yyyy`nMMM"}}, ComputerName, DisplayName, Commandlet, ModuleVersion, Event, FullCommand `
       | Export-Excel $ExcelFile @ExportExcelOptions -Passthru

    $PivotTableOptions = @{
        ExcelPackage        = $Excel
        PivotTableName      = "ByCommand"
        SourceWorkSheet     = $Excel.Sheet1
        PivotRows           = "Commandlet"
        PivotData           = @{"Commandlet" = "Count"}
        PivotColumns        = @("DateYM")
        PivotTotals         = "Both"
        PivotTableStyle     = "Light16"
        }
    $pt = Add-PivotTable @PivotTableOptions -PassThru

```
That code creates this: 

![image](https://user-images.githubusercontent.com/8366040/201506050-e48dd87e-f0c6-47c0-874f-31023fe73a9f.png)

I found this teaser in [v5.3](https://github.com/dfinke/ImportExcel/blob/7c473d1fb6faccf7722e7c3219af891c483d3b4c/README.original.md) where it talks about sort on pivot tables, but I can't figure it out.

Did I miss something in the docs?
Or is this even possible?

**Comment by dfinke** [comment]
Moved to issues https://github.com/dfinke/ImportExcel/issues/1307
